src/Apps/Artwork/Components/__stories__/OtherAuctions.story.tsx

```diff
@@ -36,9 +39,7 @@ storiesOf("Apps/Artwork Page/Components/OtherAuctions", module).add(
   () => {
     return (
       <Section title="Responsive Other Auctions">
-        <OtherAuctions>
-          {auctions.map(auction => <AuctionCard {...auction} />)}
```

Comment:
yea, `AuctionCard` here references the 'dumb' component, and not the fragment container. So this whole story just was storying out dumb components with fixture data.

Now, `OtherAuctions` references the fragment container for `AuctionCard`, so you need the usual Relay storybook setup.

Reply:
Btw... Wondering why this component even part of this 'other works' container. Maybe it should be refactored even further to include it's own fragment and render itself. More like RelatedArtist does. I guess it can even just take sale Id and look up related sales.

Was not sure if I understand the drawbacks of that possibility. What do you think @mzikherman ?